test/C/test_constr_matrix.jl

```diff
@@ -0,0 +1,36 @@
+using Gurobi, Test
+
+@testset "constr_matrix" begin
+
+    simple_model_env = Gurobi.Env()
+    setparam!(simple_model_env, "OutputFlag", 0)
+
+    simple_model = Gurobi.Model(simple_model_env, "simple_mip", :maximize)
+
+    add_ivar!(simple_model, 0.0, 0.0, Inf)  # p1
+    add_ivar!(simple_model, 0.0, 0.0, Inf)  # p2
+    add_ivar!(simple_model, 0.0, 0.0, Inf)  # p3
+    add_cvar!(simple_model, 1. , 0.0, Inf) # z
```

Comment:
Sorry, I wasn't clear enough. I mean add trailing zeros for all floating point constants, not just `0.`.